Gemfile

```diff
@@ -5,3 +5,4 @@ gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+gem "rubocop", require: false
```

Comment:
We can group it to being only available in `dev` and `test` like this:
```ruby
gem "rubocop", group: [:test, :development]
```

WDYT?